packages/ecommerce-app-base/src/AppConfig/OrchestrationEapNote/OrchestrationEapNote.tsx

```diff
@@ -13,19 +13,14 @@ const styles = {
 
 interface Props {
   name: string;
-  hideOrchestrationEapNote: boolean;
+  onClose: () => void;
 }
 
-export const OrchestrationEapNote = ({ hideOrchestrationEapNote, name }: Props) => {
-  const [, setHideOrchestrationEapNote] = useState(false);
+export const OrchestrationEapNote = ({ name, onClose }: Props) => {
   const title = `The ${name} app supports External references`;
+
   return (
-    <Note
-      style={{ display: hideOrchestrationEapNote ? 'none' : 'grid' }}
```

Comment:
Why is this no longer needed?

Reply:
Because we are checking this [here](https://github.com/contentful/apps/pull/5542/files#diff-debf4298077092d6fcc4bebd9daa39f85e1c9c77cc5de1b5acf8e5bcd37a468cR262)